documentation/assemblies/oauth/assembly-managing-policies-permissions-keycloak.adoc

```diff
@@ -0,0 +1,23 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-managing-policies-permissions-keycloak_{context}"]
+= Managing policies and permissions in Keycloak Authorization Services
+
+:context: assembly-keyword
+
+[role="_abstract"]
+This section describes the relationship between Kafka's authorization model and the authorization model used in Keycloak Authorization Services.
+It explains how to grant Kafka permissions to user accounts or service accounts using Keycloak Authorization Services rules.
+Keycloak Authorization Services is part of Keycloak.
+The mappings are used when granting permissions to access Kafka.
```

Comment:
We haven't talked about mappings. Can we say something about that?

Reply:
"To grant permissions to access Kafka, you can map Keycloak Authorization Services objects to Kafka resources by creating an _OAuth client specification_ in Keycloak."

Detailed information is given the later module, "Map Keycloak Authorization Services to the Kafka authorization model".